server/src/completions/spdx-licenses.ts

```diff
@@ -54,21 +54,27 @@ const spdxLicensesFileContent = fs.readFileSync(spdxLicensesPath, 'utf8')
 export const spdxLicensesCollection = JSON.parse(spdxLicensesFileContent) as SpdxLicenseCollection
 export const spdxLicenses = spdxLicensesCollection.licenses
 
-export const licenseCompletionItems: CompletionItem[] = spdxLicenses.map((license) => (
-  {
-    label: license.licenseId,
-    kind: CompletionItemKind.Value,
-    deprecated: license.isDeprecatedLicenseId === true,
-    labelDetails: {
-      description: spdxLicenseDescription
-    },
-    documentation: 'Loading...',
-    data: {
-      source: spdxLicenseDescription,
-      payload: license
-    } satisfies CompletionItemData
-  }
-))
+export const getLicenseCompletionItems = (rangeOfText: Range): CompletionItem[] => {
```

Comment:
Bonus: have documentation hover on licenses, possibly in a follow up PR.

Reply:
I had the same idea in mind!

Lets do it in a further PR